php7/php7.3-fpm-nginx/nginx.vh.default.conf

```diff
@@ -76,4 +76,16 @@ server {
 	location ~ /\.ht {
 		deny  all;
 	}
+
+    # Don't allow direct access to PHP files in the vendor directory.
+    location ~ /vendor/.*\.php$ {
```

Comment:
This (blocking `vendor` directory) seems only to be added to some of the configs, shall it be added to all configs. For apache config, this may be done with
```
<LocationMatch "/vendor/.*\.php$">
     Require all denied
</LocationMatch>
```